rca/project_styleguide/templates/patterns/pages/forms/form_page.html

```diff
@@ -67,24 +67,31 @@ <h1 class="title-area__heading heading heading--display-two">
                                                 {{ field }}
                                                 <label for="{{ field.id_for_label }}" class="form-item__label {% if field.field.required %}required{% endif %}">{{ field.label }} {% if field.field.required %} <span class="form-item__required">*</span>{% endif %}</label>
                                             {% else %}
-                                                <label for="{{ field.id_for_label }}" class="form-item__label {% if field.field.required %}required{% endif %}">{{ field.label }} {% if field.field.required %} <span class="form-item__required">*</span>{% endif %}</label>
-                                                {% if widget_type == 'checkbox_select_multiple'%}<p class="form-item__instruction">Select all that apply</p>{% endif %}
+                                                <{% if widget_type == 'checkbox_select_multiple' %}legend{% else %}label for="{{ field.id_for_label }}"{% endif %} class="form-item__label {% if field.field.required %}required{% endif %}">
+                                                    {% if field_type == 're_captcha_field' %}
+                                                        reCaptcha
+                                                    {% else %}
+                                                        {{ field.label }}
+                                                    {% endif %}
+                                                    {% if field.field.required %}<span class="form-item__required">*</span>{% endif %}
```

Comment:
should this be aria-hidden? not sure if a screen reader picks up symbols

Reply:
Wasn't sure, but now you've said it screen readers will take required from the input param, so i'll add it in